thriftbp/client_pool.go

```diff
@@ -167,7 +167,7 @@ type ClientPoolConfig struct {
 	// When BreakerConfig is non-nil,
 	// a breakerbp.FailureRatioBreaker will be created for the pool,
 	// and its middleware will be set for the pool.
-	BreakerConfig *breakerbp.Config
+	BreakerConfig *breakerbp.Config `yaml:"circuitBreaker"`
```

Comment:
in other yaml tags we usually only change the capitalization, never the word itself, so this should be `breakerConfig` instead. or you can rename the field to `CircuitBreaker`.